pages/information/hardware/usage.md

```diff
@@ -0,0 +1,89 @@
+
+Factors influencing usage emissions include the device's energy efficiency, usage patterns, life span, and the [carbon intensity](/glossary#carbon-intensity) of the energy grid powering it.
+
+The ICT sector's electricity consumption was estimated at 1,183 TWh in 2022, accounting for 4.7 percent of the world's total electricity use in 2021[^worldbank]. With the accelerating pace of digitalisation and the increasing reliance on ICT services and infrastructure, the energy demands of the sector are projected to rise further in the coming years. Minimising usage emissions through efficiency improvements and cleaner energy sources is therefore crucial, in conjunction with reducing embodied emissions, to comprehensively reduce the greenhouse gas impact of a product or device across its complete life cycle.
+
+{% include linkedHeading.html heading="A High-Level Approach for Estimating Usage Emissions" level=2 %}
+
+At a high level, the energy consumption of a device can be estimated using the following formula:
+
+$$ E = P \times t $$
+
+Where
+- E is energy consumption (kWh).
+- P is average power draw (kW).
+- t is time i.e. duration of usage (hours).
+
+The average power draw (P) can be obtained from industry averages, manufacturer data, or actual readings, in increasing order of precision but also complexity. Organisations should balance the need for precision with practical considerations when selecting the source for P, keeping in mind the goal of effectively directing technology optimisation efforts.
+
+Estimated energy consumption can be combined with a [carbon intensity](/glossary#carbon-intensity) factor to estimate usage emissions using the following formula:
+
+$$ U = E \times C_I $$
+
+Where
+- U is usage emissions (kg CO2e).
+- E is energy consumption (kWh).
+- C<sub>I</sub> is carbon intensity (kg CO2e per kWh).
+
+For data centre infrastructure, such as server hardware, multiply the estimated usage emissions by the data centre's [Power Usage Effectiveness (PUE)](/glossary#power-usage-effectiveness-pue) to assess the overall climate impact beyond a sole focus on the hardware equipment itself. According to recent data, the global average PUE stood at 1.58 in 2023[^pue], indicating that for every unit of energy consumed by the IT equipment, an additional 0.58 units of energy were consumed by the data centre facility's overhead systems.
+
+The per device estimates can be scaled up to give a total estimate of the usage emissions across an organisation's estate **per hardware model and location**. Combined, the formula becomes:
+
+$$ U_T = N \times O $$
```

Comment:
```suggestion
$$ U_T = N \times U $$
```
Think this is a typo based on the surrounding equations?

Reply:
Yeah thanks, I hadn't changed it from operational 😄